jbpm-services/jbpm-executor/src/test/java/org/jbpm/executor/DBUnavilabilityExecutorTest.java

```diff
@@ -47,33 +46,29 @@
 
 public class DBUnavilabilityExecutorTest{
 
-    public static final Map<String, Object> cachedEntities = new HashMap<String, Object>();
+    public static final Map<String, Object> cachedEntities = new HashMap<>();
     private ExecutorService executorService;
 
+    private static Properties dsProps;
 
     private EntityManagerFactory emf = null;
 
 	private PoolingDataSourceWrapper pds;
 
-	private static Server h2Server;
-
 	@BeforeClass
-    public static void createDBServer() throws Exception {
-	    h2Server = Server.createTcpServer(new String[] { "-tcpPort", "9123" });
-	    h2Server.start();
+    public static void createDBServer() {
+        dsProps = ExecutorTestUtil.getDatasourceProperties();
+        dsProps.setProperty("url", "jdbc:h2:tcp://localhost/target/jbpm-exec-test;MVCC=TRUE");
```

Comment:
Just checking, here without setting any port, default will be used (or a random, if it's already on use), no? Because previously it was started on 9123.

Reply:
+1, I would leave the URL as it was before.

Reply:
done